examples/gcf-pubsub-vm-delete-event-handler/main.py

```diff
@@ -146,18 +207,22 @@ def delete_record(project: str, managed_zone: str, record: dict, http=None):
         managedZone=managed_zone,
         body=change)
     response = request.execute()
-    log.debug(json.dumps({
-        'message': 'END Deleting record',
+    structured_log_data = {
+        'message': "DNS RECORD DELETED",
+        'reason': "DNS_RECORD_DELETED",
         'project': project,
         'managed_zone': managed_zone,
         'record': record,
-        'response': response}))
-    log.info(json.dumps({
-        'message': "DNS RECORD DELETED",
+        'response': response
+    }
+    log.debug(json.dumps({
```

Comment:
To make it easy to correlate let's log the exact same data unstructured via execution_id as we log structured via event_id.